Resources/Textures/Structures/Machines/VendingMachines/pride.rsi/meta.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "The art belongs to Woods#1999 on discord",
```

Comment:
Did you ask woods about the license? Was CC-BY-SA 3.0 instead of 4.0 a choice?

Reply:
Nope, was my mistake out of lack of knowledge, sorry
Asked woods and updated license